src/Microsoft.VisualStudio.Editors/MyExtensibility/MyExtensibilityPropPage.vb

```diff
@@ -114,23 +114,23 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             RemoveExtension()
         End Sub
 
-        Private Sub listViewExtensions_AddExtension(sender As Object, e As EventArgs) _
-                Handles listViewExtensions.AddExtension
+        Private Sub DataGridViewExtensions_AddExtension(sender As Object, e As EventArgs) _
+                Handles DataGridViewExtensions.AddExtension
             AddExtension()
         End Sub
 
-        Private Sub listViewExtensions_ColumnClick(sender As Object, e As ColumnClickEventArgs) _
-                Handles listViewExtensions.ColumnClick
-            ListViewComparer.HandleColumnClick(listViewExtensions, _comparer, e)
+        Private Sub DataGridViewExtensions_ColumnClick(sender As Object, e As DataGridViewCellMouseEventArgs) _
+                Handles DataGridViewExtensions.ColumnHeaderMouseClick
+            'ListViewComparer.HandleColumnClick(DataGridViewExtensions, _comparer, e)
```

Comment:
This is commented because DataGridView handles sorting automatically?  If so, just remove the handler?

Reply:
Removed